### PR TITLE
Hide VAMC SideNav sub-items on mobile

### PR DESCRIPTION
**Pull request by mpelzsherman**
https://github.com/department-of-veterans-affairs/va.gov-team/issues/10997

## Description

Previously, the VAMC SideNav sub-items were not being hidden
from keyboard nav or screen readers. This could leave users
trapped in the sub-items if navigating with a keyboard, and
caused screen readers to read the sub-items even though they
were not clickable.

## Testing done
Manual + automated (unit test)

### BEFORE
tDvflclLOe

### AFTER
BOKPCMentB

## Acceptance criteria
- [x]  Navigate the sidenav without getting trapped.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by mpelzsherman** [comment]
@brangi and I paired on this PR.

**Review by brangi — APPROVED** [review]

**Review by f1337 — APPROVED** [review]
LGTM!

File: src/platform/site-wide/side-nav/components/SideNav.js [hunk 1]
```diff
@@ -14,12 +15,30 @@ class SideNav extends Component {
...
     };
   }
 
+  getDesktop = () => {
+    return window.innerWidth > 768;
+  };
+
+  updateIsDesktop = () => {
+    this.setState({ isDesktop: this.getDesktop() });
+  };
```
Comment (U-DON) [inline src/platform/site-wide/side-nav/components/SideNav.js:32]: Would `getIsDesktop` and `setIsDesktop` be more consistent for naming?

**Review by U-DON — APPROVED** [review]

File: src/platform/site-wide/side-nav/components/SideNav.js (hunk 1 above)
Comment (ncksllvn) [reply src/platform/site-wide/side-nav/components/SideNav.js:32]: This could be a really useful React hook!

**Pull request merged**